### DISCUSSION TITLE
Variable word lengths?

### DISCUSSION THREAD

**Discussion by raderj** [octokatherine/word-master#84] [Ideas]
How about a version with seven starting spaces, allowing blanks and maybe a hyphen?

**Comment by octokatherine** [comment]
we have [an issue](https://github.com/octokatherine/word-master/issues/31) open to add different word lengths!

**Comment by kebwi** [comment]
If anyone would like to see my implementation of a Wordle variant with variable length words, check out Blurdle at https://keithwiley.com/blurdle . I started that project out as a fork of octokatherine's repo, but quickly pivoted to a from-scratch Brython-based implementation instead. I just don't have the patience to try to get Javascript to install or build correctly. After I banged the dependencies against the wall unsuccessfully, I just gave up. Brython is super easy to get up and running. Anyway, enjoy Blurdle. The source is all in the webpage if you want to see it.

Cheers!
